components/atoms/LanguagePill/LanguagePill.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import Image from "next/legacy/image";
+
+import JavascriptIcon from "img/icons/☕️.svg";
+import ReactIcon from "/img/icons/⚛️.svg";
+import PythonIcon from "/img/icons/🐍.svg";
+import AIIcon from "/img/icons/🤖.svg";
+import MLIcon from "/img/icons/🧠.svg";
+import RustIcon from "/img/icons/🦀.svg";
+
+interface LanguagePillProps {
+  topic: "react" | "javascript" | "python" | "ML" | "AI" | "rust" | string;
+  classNames?: string;
+  onClick?: () => void;
+}
+const LanguagePill = ({ topic, classNames, onClick }: LanguagePillProps) => {
+  const renderTopicIcon = (name: string) => {
```

Comment:
Let's make this a map instead so we don't have this growing list of nested ternaries.

```ts
const iconMap = {
  react: ReactIcon,
  // others
}

return iconMap[name] || ""
```